cli/cmd/root.go

```diff
@@ -0,0 +1,79 @@
+package cmd
```

Comment:
Could you add a `version` sub-command or a `--version` top-level flag?

Reply:
Added.

Reply:
Thanks!